homeassistant/util/unit_system.py

```diff
@@ -45,7 +45,7 @@
 class BaseUnitSystem(StrEnum):
     """Base unit system."""
 
-    INTERNATIONAL = "international"
+    SI = "si"
```

Comment:
We currently don't have SI are we introducing that?
Are we dropping metric?

Reply:
You are right, I have put it back as METRIC